app/assets/javascripts/statistics.es6

```diff
@@ -0,0 +1,43 @@
+export class Statistics { 
+  constructor(collection_data, work_data) {
```

Comment:
I wonder if this might be cleaner as, say, `Statistics(data, tab_content)`? I was initially surprised at how it was wired into the view (expecting the reference to the DOM element would be explicit there..).